test/parallel/taskPar/vass/with-false.good

```diff
@@ -1 +1,2 @@
 with-false.chpl:3: syntax error: near 'false'
```

Comment:
This second syntax error being issued is annoying, and probably because you do a special `error` nonterminal. I doubt you can do anything but I just wanted to register my discontent.

Reply:
It is exactly as you describe and I have similar discontent, but there wasn't much to do about it.